### ISSUE TITLE
[HOLD for payment 2024-02-15] Web - Report - Blank Concierge report is displayed on back navigation from search page

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#36035]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.38.0
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Sign up with a new account
2. Click on search bar
3. Click back button

## Expected Result:
Concierge report should be opened and report shouldn't be blank

## Actual Result:
Blank Concierge report is displayed

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/93399543/bab2058d-6991-4a2b-8abd-3a755bfa4dbb

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kbecciv** [comment]
We think that this bug might be related #vip-vsb
CC @quinthar

**Comment by situchan** [comment]
regression from https://github.com/Expensify/App/pull/35058
cc: @lukemorawski

**Comment by tgolen** [comment]
@lukemorawski Is this something you will be able to fix ASAP?

**Comment by situchan** [comment]
The root cause is same as other holding issues like #35965, #35969
@trjExpensify should we hold this as well?

**Comment by tgolen** [comment]
Yeah, I think we should. I'll put this on HOLD.

**Comment by situchan** [comment]
cc: @adamgrzybowski to put in [your PR](https://github.com/Expensify/App/issues/36050)

**Comment by trjExpensify** [comment]
Makes sense! @adamgrzybowski let us know if you disagree though.

**Comment by hayata-suenaga** [comment]
this issue might be partially related to the issue on which Adam is working on.

instead of closing this, I'll keep this issue open with HOLD as Tim suggested.

**Comment by tgolen** [comment]
Dropping to a daily while this is on HOLD

**Comment by lukemorawski** [comment]
proposal from @GandalfGwaihir fixes on the issue on the `SearchPage`. I can post quick PR for that.

**Comment by lukemorawski** [comment]
@GandalfGwaihir awesome. I would very thankful if you could post that quick fix. Seem like a bigger thing just popped in for me :)

**Comment by lukemorawski** [comment]
No idea @tgolen ⬆️ ?

**Comment by Julesssss** [comment]
Hi @lukemorawski please raise the PR now, I'll review it.

**Comment by Julesssss** [comment]
Ah sorry, I was confused. @GandalfGwaihir would be able to raise the PR?

**Comment by mountiny** [comment]
@lukemorawski Just to be clear, there should not really be bigger things than deploy blockers from PRs you have authored. This should be your top priority so we can unblock deploy.

**Comment by lukemorawski** [comment]
@mountiny Got ya! @GandalfGwaihir will post that PR!

**Comment by adamgrzybowski** [comment]
Hey, checkout this PR https://github.com/Expensify/App/pull/36050 I think it should be fixed

**Comment by Julesssss** [comment]
> Hey, checkout this PR #36050 I think it should be fixed

Thanks for sharing. But as we're going to have to cherry-pick any fix to staging I'm a bit hesitant to rely on that relatively complex PR. 

I'd much rather stick to the simpler PR raised by @lukemorawski [here](https://github.com/Expensify/App/pull/36141).

**Comment by Julesssss** [comment]
Fix CP'd to staging. I updated the checklist and removed the blocker label.

**Comment by mountiny** [comment]
No payments required
